### ISSUE TITLE
Text tool is not working properly

### ISSUE THREAD

**Issue by ebarranco** [synfig/synfig#631]
**Synfig version & platform**:
master 1.3.10
appimage 1.2.2
source 1.2.2 

**Issue description**:

Text tool is not working properly, you can't see the content of the layer created.

**Comment by ebarranco** [comment]
You can see a video about the [problem here.](https://www.dropbox.com/s/c2qa7q4ro9voaby/2018-09-24%2023-56-43.mov?dl=0)

**Comment by morevnaproject** [comment]
I can see from the video that you have Draft Mode enabled. The Draft Mode does not renders Text Layers at the moment.
![screenshot_001](https://user-images.githubusercontent.com/332868/47546657-7761a600-d91c-11e8-9e62-d1b26162fd81.png)

Please disable Draft Mode to see the text layer.

**Comment by ebarranco** [comment]
Yeah, I noticed a little later. That's it. No text layer in draft mode.

**Comment by morevnaproject** [comment]
I have renamed the issue. ^__^

**Comment by jmb** [comment]
I'm not sure if it's related or a separate issue, but on both 1.2.2 and 1.3.13 on the Mac, text layers don't seem to show at all and don't render.
https://forums.synfig.org/t/text-not-showing-at-all-mac/10865

**Comment by morevnaproject** [comment]
Resolved in Synfig version 1.5.1 - https://www.synfig.org/2021/10/30/synfig-studio-1-5-1-released/
